http/handler_acp.go

```diff
@@ -75,44 +75,110 @@ func (s *acpHandler) AddDocActorRelationship(rw http.ResponseWriter, req *http.R
 	responseJSON(rw, http.StatusOK, addDocActorRelResult)
 }
 
-func (h *acpHandler) bindRoutes(router *Router) {
-	successResponse := &openapi3.ResponseRef{
-		Ref: "#/components/responses/success",
+func (s *acpHandler) DeleteDocActorRelationship(rw http.ResponseWriter, req *http.Request) {
+	db, ok := req.Context().Value(dbContextKey).(client.DB)
+	if !ok {
+		responseJSON(rw, http.StatusBadRequest, errorResponse{NewErrFailedToGetContext("db")})
```

Comment:
suggestion: panic or internal server error (500) would be better imo. Bad request means the user did something wrong.

Reply:
Discussed on discord will resolve in:
https://github.com/sourcenetwork/defradb/issues/3103